### PR TITLE
login feature implemented

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/core/selenium/WebDriverConfig.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import java.util.Properties;
+
+/**
+ * WebDriverConfig class.
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class WebDriverConfig {
+    InputStream input;
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/WebDriverConfig.java]: Access modifiers missing.

File: src/main/java/core/utils/AmazonUtils.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+/**
+ * AmazonUtils class.
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class AmazonUtils {
+    private WebDriver driver;
+    WebDriverManager webDriverManager;
+    private static String url = "https://www.calculadora.org/";
```
Comment (MelissaRoman06) [inline src/main/java/core/utils/AmazonUtils.java]: Calculadora URL should not be used.

File: src/main/java/core/selenium/webdriver/IBrowser.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+/**
+ * IBrowser class.
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public interface IBrowser {
+
+    public void connection();
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/webdriver/IBrowser.java]: Documentation missing.

File: src/test/resources/Login.feature [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Feature: Login in an account
+
+  Scenario: login in amazon
+    Given user go to login page
+    When user put username and password
+    Then user login succefuly
```
Comment (MelissaRoman06) [inline src/test/resources/Login.feature]: Typo error. Misspelling.

File: src/main/java/core/selenium/WebDriverConfig.java (hunk 1 above)
Comment (titor-oopart) [reply src/main/java/core/selenium/WebDriverConfig.java]: done

File: src/main/java/core/utils/AmazonUtils.java (hunk 2 above)
Comment (titor-oopart) [reply src/main/java/core/utils/AmazonUtils.java]: done

File: src/main/java/core/selenium/AmazonConfig.java [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+import java.util.Properties;
+
+/**
+ * AmazonConfig class.
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class AmazonConfig {
+
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/AmazonConfig.java]: unnecessary empty line

File: src/main/java/core/selenium/AmazonConfig.java [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    private InputStream input;
+    private Properties properties;
+    private String username;
+    private String password;
+    private String url;
+
+
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/AmazonConfig.java]: unnecessary empty line

File: src/main/java/core/selenium/WebDriverConfig.java [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+import java.util.Properties;
+
+/**
+ * WebDriverConfig class.
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class WebDriverConfig {
+
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/WebDriverConfig.java]: unnecessary empty line

File: src/main/java/core/selenium/webdriver/Chrome.java [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+     * Method for connect with chrome.
+     *
+     * @return driver - WebDriver.
+     */
+    public static WebDriver driverConnection() {
+        ChromeDriverManager.getInstance().version("76.0.3809.126").setup();
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.addArguments("--incognito");
+        driver = new ChromeDriver(chromeOptions);
+        driver.get("https://www.amazon.com//");
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/webdriver/Chrome.java]: I think the url should be given in the page class.

File: src/main/java/core/selenium/webdriver/Firefox.java [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+    public static WebDriver driverConnection() {
+        FirefoxDriverManager.getInstance().version("0.24.0").setup();
+        FirefoxOptions firefoxOptions = new FirefoxOptions();
+        firefoxOptions.addArguments("--incognito");
+        driver = new FirefoxDriver(firefoxOptions);
+        driver.get("https://www.calculadora.org/");
+        driver.manage().window().maximize();
+        return driver;
+    }
+
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/webdriver/Firefox.java]: Unnecessary empty lines

File: src/main/java/core/utils/AmazonUtils.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/core/utils/AmazonUtils.java]: Maybe this get () should be outside the web driver manager.

**Pull request merged**